chapters/sampling-randomization-power.tex

```diff
@@ -1,529 +1,782 @@
+This includes both how a sample is representative to the population studied,
+or how the counterfactual observations in experimental design are statistically indistinguishable
+from the treatment observations.
+The second half concludes with a discussion of power calculations and randomization inference,
+and how both are important tools to make optimal choices when planning data work.
+
+
+\end{fullwidth}
+
+%-----------------------------------------------------------------------------------------------
+
+\section{Translating research design to master data}
+
+In most projects, more than one dataset is needed to answer the research question.
+These could be multiple survey rounds,
+integration of different data sources (e.g. administrative data,
+web scraping, implementation monitoring, etc)
+or complex combinations of the above.
+For example, you may have different levels of \textbf{unit of observation}\sidenote{
+	The \textbf{unit of observation} is the unit at or for which data is collected. See
+	\url{https://dimewiki.worldbank.org/Unit\_of\_Observation}},
+or the units of observation you do have (or their subunits)
+may vary from round to round.
+
+However your study is structured, you need to know how to link all the datasets you obtain
+and analyze the relationships between the units that appear in them
+to answer all your research questions.
+You might think that you are able to keep all of the relevant details in your head,
+but your whole research team is unlikely to have the same understanding,
+at all times, of all the datasets required.
+The only way to make sure that the full team shares the same understanding
+is to create \textit{master datasets} and a \textit{data map}.
+
+
+\subsection{Creating master datasets and a data map}
+
+A \textbf{master dataset}\sidenote{
+	\url{https://dimewiki.worldbank.org/Master\_Data\_Set}}
+\index{master datasets}
+details all project-wide time-invariant information
+about all units that could appear in the data
+and their relationship to the research design,
+like sampling and treatment status once those are determined.
+Solving all of these questions before you acquire any data,
+and making sure that the full research team knows where to find this information,
+will save a ton of time later on,
+and will increase the quality of your research.
+
+You should create a master dataset
+for each unit of observation
+important to the research.
+A unit used for any significant research activity,
+like data collection or data analysis, 
+must always be considered an important unit of observation.
+Therefore, any unit
+that will be used in sampling or treatment assignment,
+must have a master dataset,
+and that master dataset -- not field data --
+should be used when sampling or assigning treatment.
+
+You also need to record how all datasets for each unit of observation
+will link or merge with each other as needed.
+This linking scheme is called a \textbf{data map}\sidenote{
```

Comment:
It would be helpful to make this more concrete with some suggestions of how to make a data map (and/or examples), specifically what software one would use to actually create a data map. i've seen anything ranging from scribbles on the back of a piece of paper, to nicely formatted Word diagrams, to Excel sheets...

Reply:
My planning for the wiki page is that it will include examples, and I will provide an Excel sheet template. Might include a flow chart example as well.

Reply:
What are you thinking of as a data map Luiza? can you give an example?

Reply:
i think our conversation today clarified this enough to close the conversation here, though i'm sure we'll return to it when we have a rewrite.